pkg/security/probe/fields_resolver.go

```diff
@@ -0,0 +1,246 @@
+//go:build linux
+// +build linux
+
+// Code generated - DO NOT EDIT.
+
+package probe
+
+// ResolveFields resolves all the fields associate to the event type. Context fields are automatically resolved.
+func (ev *Event) ResolveFields() error {
+	// resolve context fields that are not related to any event type
+	_ = ev.ResolveContainerID(&ev.ContainerContext)
+	_ = ev.ResolveContainerTags(&ev.ContainerContext)
+	_ = ev.ResolveProcessArgs(&ev.ProcessContext.Process)
+	_ = ev.ResolveProcessArgsTruncated(&ev.ProcessContext.Process)
+	_ = ev.ResolveProcessArgv(&ev.ProcessContext.Process)
+	_ = ev.ResolveProcessArgv0(&ev.ProcessContext.Process)
+	_ = ev.ResolveProcessCreatedAt(&ev.ProcessContext.Process)
+	_ = ev.ResolveProcessEnvp(&ev.ProcessContext.Process)
+	_ = ev.ResolveProcessEnvs(&ev.ProcessContext.Process)
+	_ = ev.ResolveProcessEnvsTruncated(&ev.ProcessContext.Process)
+	_ = ev.ResolveFileFieldsGroup(&ev.ProcessContext.Process.FileFields)
+	_ = ev.ResolveFileFieldsInUpperLayer(&ev.ProcessContext.Process.FileFields)
+	_ = ev.ResolveFileFieldsUser(&ev.ProcessContext.Process.FileFields)
+
+	// resolve event specific fields
+	switch ev.GetEventType().String() {
+
+	case "bpf":
+		_ = ev.ResolveHelpers(&ev.BPF.Program)
+		break
```

Comment:
that is a `C` thing :), you can remove the `break`

Reply:
👍🏻